Library/Homebrew/cmd/test-bot.rb

```diff
@@ -520,7 +521,7 @@ def formula formula_name
               test "brew", "uninstall", "--force", *unchanged_build_dependencies
               unchanged_dependencies -= unchanged_build_dependencies
             end
-            test "brew", "install", bottle_filename
+            test "brew", "install", "--force-bottle", bottle_filename
```

Comment:
My concern is that as-is this completely changes the behaviour of the bot and, unless I'm mistaken, stops it from verifying bottle checksums any more. I'm open to any solution that deals with that such as adding a new `--build-bottle` argument to the bot.